### DISCUSSION TITLE
[Bug] The link to project after application is wrong.

### DISCUSSION THREAD

**Discussion by jaiakash** [community/community#58200] [GitHub Education] [answered]
### Hi everyone,

So i was applying to the Dive project on the Octernship page. But after the application, the link generated to repo is unreachable. Moreover there is no creation of private repo by Github Classroom bot as with other projects.

https://github.com/community/community/assets/33419526/6b27b7d6-3e1e-4317-8749-03a7ed772a6b

**Comment by Arindam-7** [comment]
I'm on the same boat, did you happen to find any workaround for this problem?

**Reply by jaiakash** [reply]
Not yet

**Comment by dibenzepin** [comment]
I decided to go looking for my own template repo (I didn't get redirected to it either and have been working on a fork) and I just found it by going to the account that posted the Octernship ([@rustdesk_org](https://github.com/rustdesk-org) in my case) and looking through their repos...then I saw one that had my username in it

**Comment by ashwinexe** [comment, marked as answer]
Thanks for reporting this @jaiakash. We will take a look into this. 
You also get an email after applying to Octernships project. You can access your private repo from the email :)

**Reply by jaiakash** [reply]
Thanks for that.
